Week-3/assignment3.4/closureStack.js

```diff
@@ -0,0 +1,24 @@
+function stack() {
+  let items = [];
+  function add(arg) {
+    items.push(arg);
+    console.log(items);
+  }
+  function get() {
+    console.log(items.pop());
+  }
+  return [add, get];
+}
+const [add, get] = stack();
+
+//Using closure we have scoped items
+//variable in a closure.
+//items variable cant be used outside the scope
+add(5);
```

Comment:
try using push and pop instead of add and get. That is the designated method names for stack. The functionality is fine